### PR TITLE
feat: change screen share constrains [WPB-9950]

### PR DESCRIPTION
**Pull request by EnricoSchw**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-9950" title="WPB-9950" target="_blank">StoryWPB-9950</a>  [AVS] Change Screen Share constrains for better Screen Share quality in 1:1 Call
  </td></table>
  <br />
 

## Description

With this pull request, we are increasing the resolution for screen sharing. To do this, we had to remove the bandwidth limitation in [wire-avs](https://github.com/wireapp/wire-avs/releases/tag/9.8.14) for One to one call. Because Chrome and Firefox can handle this automatically, so it is no necessary.

## Checklist

- [x] PR has been self reviewed by the author;
- [x] Hard-to-understand areas of the code have been commented;
- [x] If it is a core feature, unit tests have been added;

### CODE REVIEW

File: src/script/media/MediaConstraintsHandler.ts [hunk 1]
```diff
@@ -57,28 +57,19 @@ export class MediaConstraintsHandler {
           DESKTOP_CAPTURER: {
             mandatory: {
               chromeMediaSource: 'desktop',
-              maxHeight: 1080,
-              minHeight: 1080,
             },
           },
           DISPLAY_MEDIA: {
             frameRate: 5,
```
Comment (tlebon) [inline src/script/media/MediaConstraintsHandler.ts:63]: will this value be superceded in a 1:1 call?

File: src/script/media/MediaConstraintsHandler.ts [hunk 2]
```diff
@@ -57,28 +57,19 @@ export class MediaConstraintsHandler {
           DESKTOP_CAPTURER: {
             mandatory: {
               chromeMediaSource: 'desktop',
-              maxHeight: 1080,
-              minHeight: 1080,
```
Comment (tlebon) [inline src/script/media/MediaConstraintsHandler.ts:61]: can you also discuss removing the height values now?

File: src/script/media/MediaConstraintsHandler.ts (hunk 1 above)
Comment (EnricoSchw) [reply src/script/media/MediaConstraintsHandler.ts:63]: Yes, and in a conference call too.

By removing this value, the device determines the resolution. If you have a high resolution screen, this is it. WebRTC then reduces the resolution according to the bandwidth. This works well in 1:1 call. In Conference Call WebRTC can not reduces the resolution, currently.

File: src/script/media/MediaConstraintsHandler.ts (hunk 2 above)
Comment (EnricoSchw) [reply src/script/media/MediaConstraintsHandler.ts:61]: By removing this value, the device determines the resolution. If you have a high resolution screen, this is it. WebRTC then reduces the resolution according to the bandwidth. This works well in 1:1 call. In Conference Call WebRTC can not reduces the resolution, currently.

File: src/script/media/MediaConstraintsHandler.ts (hunk 1 above)
Comment (EnricoSchw) [reply src/script/media/MediaConstraintsHandler.ts:63]: Usually not
In our test, the value was exceeded briefly, but that is probably due to the sending buffer. But the value hardly increased noticeably.

**Review by tlebon — APPROVED** [review]

**Pull request merged**